src/styles/layout.scss

```diff
@@ -530,3 +530,21 @@ details {
   display: flex;
   justify-content: center;
 }
+
+.resources-logos {
+  img {
+    width: 100%;
+  }
+
+  td {
+    padding: 20px;
+  }
+}
+
+.bg-white {
+  background-color: #fff !important;
```

Comment:
Can you be more specific and not use `!important`? If not add a comment on why

Reply:
Removed that. I was using the same style as the nodejs src. Removed that,